source/features/swap-branches-on-compare.tsx

```diff
@@ -18,7 +18,7 @@ function init(): void {
 		references.unshift(select('.branch span')!.textContent!);
 	}
 
-	const icon = select('.octicon-arrow-left')!;
+	const icon = select('.range-editor .octicon-arrow-left')!;	
```

Comment:
```suggestion
	const icon = select('.range-editor .octicon-arrow-left')!;
```

You left an extra tab